src/Admin/index.php

```diff
@@ -39,6 +39,7 @@
     <?php include("../../includes/editUserModal.php"); ?>
     <?php include("adminModal.php"); ?>
     <script type="text/javascript" src="../Script/admin.js"></script>
+    <script type="text/javascript" src="../Script/public/admin.js"></script>
```

Comment:
Do script type="module" for all as eventually every single JS file will use modules